src/coreclr/jit/codegenlinear.cpp

```diff
@@ -1535,7 +1535,7 @@ regNumber CodeGen::genConsumeReg(GenTree* tree)
             if (fldVarDsc->GetRegNum() == REG_STK)
             {
                 // We have loaded this into a register only temporarily
-                gcInfo.gcMarkRegSetNpt(reg);
+                gcInfo.gcMarkRegSetNpt(genRegMask(reg));
```

Comment:
There is also one here:

https://github.com/dotnet/runtime/pull/65907/files#diff-426fb8beee6e663b3fb0f11330af0ce0ce21ff22a8ea819f39d12a95d9d6cffcL1433

Reply:
I just noticed it.

Your link didn't work; can you check?

Reply:
Its on line 1433

Reply:
ah, found it. I'll fix both, but I'm going to subsume it into #65916